aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -561,7 +562,27 @@ In the following *appsettings.json* example:
 }
 ```
 
-An alternative to using **Path** and **Password** for any certificate node is to specify the certificate using certificate store fields. For example, the **Certificates** > **Default** certificate can be specified as:
+Schema notes:
+
+* Endpoints names are case-insensitive. For example, `HTTPS` and `Https` are valid.
+* The `Url` parameter is required for each endpoint. The format for this parameter is the same as the top-level `Urls` configuration parameter except that it's limited to a single value.
+* These endpoints replace those defined in the top-level `Urls` configuration rather than adding to them. Endpoints defined in code via `Listen` are cumulative with the endpoints defined in the configuration section.
+* The `Certificate` section is optional. If the `Certificate` section isn't specified, the defaults defined in earlier scenarios are used. If no defaults are available, the server throws an exception and fails to start.
```

Comment:
What earlier scenarios? Specify them or link to them. Also what exception?

Reply:
What is an earlier scenario? The style guide prefers previous, but you need to be specific, such as

 If the `Certificate` section isn't specified, the defaults, defined previously, are used.